installation/proc-add-custom-app-config-file-ocp.adoc

```diff
@@ -0,0 +1,86 @@
+----
++
+* Use the mandatory backend auth key for {product} to reference an environment variable defined in an {ocp-short} Secret.
+* Set the external URL of your {product} instance in the `app.baseUrl`, `backend.baseUrl` and `backend.cors.origin` fields of the application configuration. By default, the URL is similar to the following example:
+`pass:c[https://backstage-<CUSTOM_RESOURCE_NAME>-<NAMESPACE_NAME>.<OPENSHIFT_INGRESS_DOMAIN>;]`.
+** You can use the `oc get ingresses.config/cluster -o jsonpath='{.spec.domain}'` command to display your ingress domain. If you want to use a different host or sub-domain, customize the `Custom Resource spec.application.route field` and adjust the application configuration accordingly.
+
+[NOTE]
+You are responsible for protecting your {product} installation from external and unauthorized access. Manage the backend auth key like any other secret. Meet strong password requirements, do not expose it in any configuration files, and only inject it into configuration files as an environment variable.
+
+.Prerequisites
+* You have an active {ocp-brand-name} account.
+
+.Procedure
+. From the *Developer* perspective, select the *ConfigMaps* tab.
+. Click *Create ConfigMap*.
+. Select the *YAML view* option in *Configure via* and make the changes to the file, if necessary.
+. Click *Create*.
+. Select the *Secrets* tab.
+. Click *Create Key/value Secret*.
+. Name the secret `secrets-rhdh`.
+. Add a key named `BACKEND_SECRET` and a base64 encoded string as a value. Use a unique value for each {product} instance. For example, you can use the following command to generate a key from your terminal:
++
+[source,yaml]
+----
+node -p 'require("crypto").randomBytes(24).toString("base64")'
+----
+
+. Click *Create*.
+. Select the *Topology* tab.
+. Click the overflow menu for the {product} instance that you want to use and select *Edit Backstage* to load the YAML view of the {product} instance.
++
+image::rhdh/operator-install-2.png[]
+
+. Add the `spec.application.appConfig.configMaps` and `spec.application.extraEnvs.secrets` fields to the Custom Resource. For example:
++
+[source, yaml]
+----
+spec:
+  application:
+    appConfig:
+      mountPath: /opt/app-root/src
+      configMaps:
+         - name: app-config-rhdh
+    extraEnvs:
+      secrets:
+         - name: secrets-rhdh
+    extraFiles:
+      mountPath: /opt/app-root/src
+    replicas: 1
+    route:
+      enabled: true
+  database:
+    enableLocalDb: true
+----
+. Click *Save*.
+. Navigate back to the *Topology* view and wait for the {product} pod to start.
+. Click the *Open URL* icon to start using the {product} platform with the new configuration changes.
+
+.Additional resources
+* For more information about roles and responsibilities in {product-short}, see xref:con-rbac-overview_{context}[Role-Based Access Control (RBAC) in {product}] in the Administration Guide for {product}.
```

Comment:
There's some guidance about additional resources in the OCP doc guidelines that I think would be useful for us here, but it's not a blocker: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-xref-attribute

Reply:
I'm fine with adding this. Can we create a separate PR so we can move forward with this one?